run.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 echo "Running Locust benchmark"
-USER_WAIT_TIME_MIN_SECONDS=0 USER_WAIT_TIME_MAX_SECONDS=0 REQUESTS_JSON="${1}" HOST="${2:-http://localhost:5000}" pipenv run locust --headless -u 1 -r 1 -t ${3:-1m} --csv=${4:-output} -L WARNING
+USER_WAIT_TIME_MIN_SECONDS=0 USER_WAIT_TIME_MAX_SECONDS=0 REQUESTS_JSON="${1}" INCLUDE_SCHEMA_URL_IN_TOKEN="${2}" HOST="${3:-http://localhost:5000}"  pipenv run locust --headless -u 1 -r 1 -t ${4:-1m} --csv=${5:-output} -L WARNING
```

Comment:
The README doesn't match this, given the order of arguments have changed. 2nd arg was the host but is now a bool.